htdocs/core/modules/modProductBatch.class.php

```diff
@@ -120,6 +120,20 @@ public function init($options = '')
 
 		$sql = array();
 
+		dolibarr_set_const($db, 'STOCK_CALCULATE_ON_BILL', 0);
+		dolibarr_set_const($db, 'STOCK_CALCULATE_ON_VALIDATE_ORDER', 0);
+		dolibarr_set_const($db, 'STOCK_CALCULATE_ON_SHIPMENT', 1);
+		dolibarr_set_const($db, 'STOCK_CALCULATE_ON_SHIPMENT_CLOSE', 0);
+		dolibarr_set_const($db, 'STOCK_CALCULATE_ON_SUPPLIER_BILL', 0);
+		dolibarr_set_const($db, 'STOCK_CALCULATE_ON_SUPPLIER_VALIDATE_ORDER', 0);
+
+        if (empty($this->reception->enabled)) {
```

Comment:
@atm-quentin  this is "$conf->reception->enabled" here !
but this check will not be executed if the "reception" module is activated or deactivated after the "productbatch" module !!

Reply:
right but same thing with cashdeks after